pkg/controller/provider/validation.go

```diff
@@ -317,3 +331,38 @@ func (r *Reconciler) inventoryCreated(provider *api.Provider) error {
 
 	return nil
 }
+
+//
+// Check for and Validate oVirt insecure option
+// and return the value of the `insecureSkipVerify` flag.
+// If flag is set, set a `ConnectionInsecure`` condition to alert users.
+func validateOVirtInsecureOption(provider *api.Provider, secret *core.Secret) bool {
+	insecure, err := insecureSkipVerify(secret)
+	if err != nil {
+		insecure = false
+	}
+
+	if insecure {
+		insecureSkipVerifyCnd := libcnd.Condition{
+			Type:     ConnectionInsecure,
+			Status:   True,
+			Reason:   InsecureSkipVerify,
+			Category: Warn,
+			Message:  "TLS is susceptible to machine-in-the-middle attacks unless custom verification is used.",
+		}
+		provider.Status.SetCondition(insecureSkipVerifyCnd)
```

Comment:
Please rewrite it to, you are not using the `insecureSkipVerifyCnd` anywhere else.
```go
provider.Status.SetCondition(
		libcnd.Condition{
			Type:     ConnectionInsecure,
			Status:   True,
			Reason:   InsecureSkipVerify,
			Category: Warn,
			Message:  "TLS is susceptible to machine-in-the-middle attacks unless custom verification is used.",
		}
)
```

Reply:
yes, makes sense to me too, using your suggestion